solr/prometheus-exporter/src/test/org/apache/solr/prometheus/collector/MetricSamplesTest.java

```diff
@@ -25,46 +28,48 @@
 import java.util.Locale;
 import java.util.Map;
 import java.util.stream.Collectors;
-
-import io.prometheus.client.Collector;
 import org.junit.Test;
 
-import static org.junit.Assert.assertEquals;
-
 public class MetricSamplesTest {
 
   private Collector.MetricFamilySamples.Sample sample(String name, Double value) {
-    return new Collector.MetricFamilySamples.Sample(name, Collections.emptyList(), Collections.emptyList(), value);
+    return new Collector.MetricFamilySamples.Sample(
+        name, Collections.emptyList(), Collections.emptyList(), value);
   }
 
-  private Collector.MetricFamilySamples samples(String metricName, Collector.Type type, Collector.MetricFamilySamples.Sample...samples) {
+  private Collector.MetricFamilySamples samples(
+      String metricName, Collector.Type type, Collector.MetricFamilySamples.Sample... samples) {
     return new Collector.MetricFamilySamples(
-        metricName,
-        type,
-        "help",
-        new ArrayList<>(Arrays.asList(samples))
-    );
+        metricName, type, "help", new ArrayList<>(Arrays.asList(samples)));
   }
 
   private void validateMetricSamples(
       List<Collector.MetricFamilySamples> allMetrics,
       String metricName,
       List<Double> expectedValues) {
 
-    Collector.MetricFamilySamples test1 = allMetrics.stream()
-        .filter(s -> s.name.equals(metricName))
-        .findFirst()
-        .orElseThrow(() -> new RuntimeException(String.format(Locale.ROOT, "Unable to find item %s", metricName)));
-
-    assertEquals(expectedValues, test1.samples.stream().map(s -> s.value).collect(Collectors.toList()));
+    Collector.MetricFamilySamples test1 =
+        allMetrics.stream()
+            .filter(s -> s.name.equals(metricName))
+            .findFirst()
+            .orElseThrow(
+                () ->
+                    new RuntimeException(
+                        String.format(Locale.ROOT, "Unable to find item %s", metricName)));
```

Comment:
Simple string concatenation probably wouldn't have wrapped

Reply:
Fixed in 6f59d6fa54dd1711174bd1e607b7c14e54fa6cd7